src/UI/StaffCraftingMenu.cpp

```diff
@@ -166,6 +166,10 @@ namespace UI
 				filterFlag = FilterFlag::Morpholith;
 			}
 			else if (allowSoulGems && object->Is(RE::FormType::SoulGem)) {
+				if (item->GetSoulLevel() == RE::SOUL_LEVEL::kNone) {
```

Comment:
I think it would read better if we just added `&& item->GetSoulLevel() != RE::SOUL_LEVEL::kNone` above.